app/services/auth.js

```diff
@@ -313,3 +220,18 @@ export default Service.extend({
     }
   }
 });
+
+function createUserRecord(store, user) {
+  const record = store.push(store.normalize('user', user));
+  const installation = store.peekAll('installation').findBy('owner.id', user.id) || null;
+  record.setProperties({ installation });
+  return record;
+}
+
+function runAfterSignOutCallbacks() {
+  afterSignOutCallbacks.forEach((callback) => {
+    callback();
+  });
```

Comment:
would `afterSignOutCallbacks.forEach(callback)` work?

Reply:
It should! Great suggestion, thanks 👍

Reply:
Ah, no, it won't. `callback` is a function that's passed to iterator, so it's undefined outside of it.

Reply:
Though, I did simplify it like this: `afterSignOutCallbacks.forEach(callback => callback());`

Reply:
Ah, right lol, I misread it. Nice!